Lib/test/test_traceback.py

```diff
@@ -2473,41 +2473,41 @@ def test_exception_group_format(self):
 
         expected = [
                     f'  + Exception Group Traceback (most recent call last):',
-                    f'  |   File "{__file__}", line {lno_g+23}, in _get_exception_group',
```

Comment:
This is using constantly `self._get_exception_group.__qualname_`, you can capture this into a variable and just use the variable for clarity in the f-strings.

Reply:
The same apply to other test files.

Reply:
I think this was the only source that could do with a local variable in multiple places?